resources/deb/etc/init.d/openhab2

```diff
@@ -71,15 +71,36 @@ fi
 # Load the VERBOSE setting and other rcS variables
 . /lib/init/vars.sh
 
+# Check to see if openHAB is currently running
+EXISTINGPID=`pgrep java -G openhab | tail -1`
+
 case "$1" in
   start)
         log_daemon_msg "Starting $DESC" "$NAME"
-        #${OPENHAB_INST_DIR}/bin/setpermissions.sh > /dev/null
-        if start-stop-daemon --start --quiet --pidfile $PIDFILE -c "${USER_AND_GROUP}" --umask 002 --background --make-pidfile --chdir ${OPENHAB_DIR} \
-                --oknodo --exec /usr/share/openhab2/runtime/bin/start
+
+        # If a process still exists but there's no PID file, then openHAB is probably still shutting down. We should proceed.
+        if [ -z "$EXISTINGPID" ] || [ ! -f $PIDFILE ];
         then
-            log_end_msg 0
+            if start-stop-daemon --start --quiet --pidfile $PIDFILE -c "${USER_AND_GROUP}" --umask 002 --background --chdir ${OPENHAB_DIR} \
```

Comment:
Ah, good idea. My only change of this line was to remove the --make-pidfile option. Since the line got shorter, it hadn't occured to me that the line was already too long to begin with!